assets/src/dashboard/app/views/editorSettings/googleAnalytics/index.js

```diff
@@ -74,16 +85,23 @@ function GoogleAnalyticsSettings({
           id="gaTrackingId"
           value={analyticsId}
           onEditCancel={handleCancelUpdateId}
-          onEditComplete={handleCompleteUpdateId}
+          onEditComplete={handleUpdateId}
           placeholder={TEXT.PLACEHOLDER}
+          error={inputError}
         />
-        <TextInputHelperText>{TEXT.CONTEXT}</TextInputHelperText>
+        <TextInputHelperText>
+          {TEXT.CONTEXT}
+          <InlineLink href={TEXT.CONTEXT_ARTICLE_LINK}>
+            {TEXT.CONTEXT_ARTICLE}
+          </InlineLink>
+          {'.'}
```

Comment:
Can we remove the dot? Since it‘s not translatable it will be more confusing with dot than without.

Reply:
we can remove it. I had added it because it should be the end of the sentence but that's pretty limiting to western languages i suppose. Consider it gone!